staging/src/k8s.io/apiextensions-apiserver/test/integration/basic_test.go

```diff
@@ -81,124 +81,163 @@ func TestClusterScopedCRUD(t *testing.T) {
 }
 
 func testSimpleCRUD(t *testing.T, ns string, noxuDefinition *apiextensionsv1beta1.CustomResourceDefinition, dynamicClient dynamic.Interface) {
```

Comment:
just to make sure, this is the exact same code of `testSimpleVersionedCRUD`? I can skip reviewing it if it is :)

Reply:
it is of course.